resources/js/components/Members.vue

```diff
@@ -371,7 +371,7 @@ export default {
       );
     },
     compositeList: function() {
-      if(this.group.include_child_groups && this.group.child_groups) {
+      if(this.group && this.group.include_child_groups && this.group.child_groups) {
```

Comment:
Good catch!

No change request, but wanted to mention that Vite (or maybe it's TS?) supports optional chaining syntax, which is nice for cases like this:
```ts
      if (this.group?.include_child_groups && this.group.child_groups) {
```

Reply:
yeah, for whatever reason that syntax doesn't read well in my head so I don't usually do it. Can try to make the switch for consistency though.